pkg/manifests/manifest_integration_test.go

```diff
@@ -0,0 +1,216 @@
+//go:build integration
```

Comment:
This file must only contain tests which perform actual downloading from the internet. All other cases belong to `manifests_test.go` instead. Also please either rename this file to plural "manifests" or the others to singular "manifest" for consistency.